### PR TITLE
Next > Adopt Angular CLI ; adopt @circlon/angular-tree-component

### PR DESCRIPTION
**Pull request by Blackbaud-AlexKingman**

### CODE REVIEW

File: .skypageslocales/resources_en_US.json [hunk 1]
```diff
@@ -0,0 +1 @@
...on":"Text for modal close button","message":"Close modal"},"skyux_modal_open_help":{"_description":"Text for modal header Open Help button","message":"Open Help"},"skyux_modal_footer_cancel_button":{"_description":"Default label text for modal cancel button","message":"Cancel"},"skyux_modal_footer_primary_button":{"_description":"Default label text for modal primary button","message":"Save"},"skyux_dropdown_context_menu_default_label":{"_description":"The label on the context menu button used for screen readers when the consumer has not specified a label","message":"Context menu"},"skyux_tab_add":{"_description":"Label for the tab add button","message":"Add tab"},"skyux_tab_close":{"_description":"Label for the tab close button","message":"Close {0} tab"},"skyux_tabs_navigator_next":{"_description":"Text displayed on a tab's next button.","message":"Next"},"skyux_tabs_navigator_previous":{"_description":"Text displayed on a tab's previous button.","message":"Previous"},"skyux_tab_open":{"_description":"Label for the tab open button","message":"Open tab"},"skyux_vertical_tabs_show_tabs_text":{"_description":"The default text for the show tabs button in mobile","message":"Tab list"}}
```
Comment (Blackbaud-SteveBrush) [inline .skypageslocales/resources_en_US.json:1]: This folder was accidentally committed.

File: .skypagestmp/docs/documentation.json [hunk 2]
```diff
@@ -0,0 +1 @@
...:116,"character":25}]},{"id":37,"name":"showSelectButtons","kind":2048,"kindString":"Method","flags":{"isPublic":true,"isExported":true},"signatures":[{"id":38,"name":"showSelectButtons","kind":4096,"kindString":"Call signature","flags":{"isExported":true},"type":{"type":"intrinsic","name":"boolean"}}],"sources":[{"fileName":"src/app/public/modules/angular-tree/angular-tree-wrapper.component.ts","line":143,"character":26}]}],"groups":[{"title":"Properties","kind":1024,"children":[15,16,17,22]},{"title":"Accessors","kind":262144,"children":[18]},{"title":"Methods","kind":2048,"children":[27,25,29,31,33,35,37]}],"sources":[{"fileName":"src/app/public/modules/angular-tree/angular-tree-wrapper.component.ts","line":33,"character":43}],"implementedTypes":[{"type":"reference","name":"AfterViewInit"}],"anchorId":"class-skyangulartreewrappercomponent"}],"groups":[{"title":"Classes","kind":128,"children":[136,58,135,14]}],"anchorIds":{"SkyAngularTreeModule":"class-skyangulartreemodule","SkyAngularTreeNodeComponent":"class-skyangulartreenodecomponent","SkyAngularTreeResourcesModule":"class-skyangulartreeresourcesmodule","SkyAngularTreeWrapperComponent":"class-skyangulartreewrappercomponent"}}
```
Comment (Blackbaud-SteveBrush) [inline .skypagestmp/docs/documentation.json:1]: Committed by accident.

File: .skypageslocales/resources_en_US.json (hunk 1 above)
Comment (Blackbaud-AlexKingman) [reply .skypageslocales/resources_en_US.json]: Removed.

File: .skypagestmp/docs/documentation.json (hunk 2 above)
Comment (Blackbaud-AlexKingman) [reply .skypagestmp/docs/documentation.json:1]: Removed.

**Review by Blackbaud-SteveBrush — APPROVED** [review]

**Pull request merged**